core/domain/platform_feature_services_test.py

```diff
@@ -102,17 +102,24 @@ def setUp(self):
         # Replace feature lists with mocked names.
         self.original_feature_list = feature_services.ALL_FEATURES_LIST
         self.original_feature_name_set = (
-            feature_services.ALL_FEATURES_NAMES_SET)
-        feature_services.ALL_FEATURES_LIST = param_name_enums
+            feature_services.ALL_FEATURES_NAMES_SET
+        )
+        # The expected type of 'ALL_FEATURES_LIST' is a list of 'PARAM_NAMES'
+        # Enum, but here for testing purposes we are providing a list of
+        # 'ParamNames' Enum. Which causes MyPy to throw an 'Incompatible types
```

Comment:
... a list of 'ParamNames' enums, which causes MyPy to ...

Reply:
Done!